synfig-studio/plugins/lottie-exporter/shapes/rectangle.py

```diff
@@ -481,27 +536,11 @@ def copy_tcb(new_waypoint, waypoint):
     new_waypoint.attrib["bias"] = str(bias)
 
 
-def switch_case(animated, i, x2_val, y2_val):
-    x_now, y_now = float(animated[i][0][0].text), float(animated[i][0][1].text)
-    x_next, y_next = float(animated[i+1][0][0].text), float(animated[i+1][0][1].text)
-    sign = lambda a: (1, -1)[a < 0]
-    x_change, y_change = 0, 0
-    if sign(x_now - x2_val) != sign(x_next - x2_val):
-        x_change = 1
-    if sign(y_now - y2_val) != sign(y_next - y2_val):
-        y_change = 1
-    
-    if x_change == 0 and y_change == 0:
-        return 0
-    elif x_change == 1 and y_change == 0:
-        return 1
-    elif x_change == 0 and y_change == 1:
-        return 2
-    elif x_change == 1 and y_change == 1:
-        return 3
-
-
 def equal_time_frame(waypoint1, waypoint2):
+    """
+    Returns True if two waypoints are present on the same frame
```

Comment:
Description is incorrect - the function returns 1 and 0

Reply:
Corrected now!